<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="pushButton_header">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>261</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>设置水平表头</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_row_set">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>设置行数</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_row_insert">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>插入行</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_row_append">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>140</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>添加行</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_init">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>261</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>初始化表格数据</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_row_delete">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>261</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>删除当前行</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_row_adjust">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>调节行高</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_column_adjust">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>220</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>调节列宽</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_read">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>261</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>读取表格内容到文本</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_editable">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>300</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>表格可编辑</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_color">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>300</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>间隔行底色</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_header">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>340</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>显示水平表头</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_vHeader">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>340</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>显示垂直表头</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_row_select">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>380</y>
          <width>97</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>行选择</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_item_select">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>380</y>
          <width>97</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>单元格选择</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>60</y>
          <width>121</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTableWidget" name="tableWidget">
        <property name="showDropIndicator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
